Firmware/LowLevel/include/config_defaults.h

```diff
@@ -0,0 +1,26 @@
+#ifndef _CONFIG_H
+#define _CONFIG_H
+
+#define BATT_ABS_MAX 28.7f
+#define BATT_ABS_MIN 21.7f
+
+#define BATT_EMPTY BATT_ABS_MIN + 0.3f
+#define BATT_FULL BATT_ABS_MAX - 0.3f
+
+#define V_BATT_CUTOFF 29.0f
+
+#define TILT_EMERGENCY_MILLIS 2500  // Time for a single wheel to be lifted in order to count as emergency (0 disable). This is to filter uneven ground.
+#define LIFT_EMERGENCY_MILLIS 100   // Time for both wheels to be lifted in order to count as emergency (0 disable). This is to filter uneven ground.
+
+#define V_CHARGE_MAX 30.0f  // Default YF-C500 max. charge voltage
+#define I_CHARGE_MAX 1.5f   // Default YF-C500 max. charge current
```

Comment:
`..._CHARGE_CUTOFF`, like in the struct? Doesn't seem to be used anywhere else.

Reply:
Yes, but not yet sure how to handle/name.
Need to clear that up when implementing the handling of the received values.
Naming of these look somehow wildly grown in past. There where some _MAX also _ABS_MAX, _EMPTY, _FULL and also some static ones before. 
I started by integrating the CUTOFF ones to make clear that those switch charging off, but I need to recheck and rethink about those because I don't wanna integrate them all in ROS. I like to use only the sensor specific ones from ROS which are Critical_Low, min, max and critical high. Initially I thought to use the critical_height @ ROS for CUTOFF @ LL and the min/max ones for full/empty... but than occurred the _ABS_ ones.
Might happen that I rename the _CUTOFFs to _ABS... later, but will check that in detail when implementing the config value handling.

Reply:
Did you see my other comment above? https://github.com/ClemensElflein/OpenMower/pull/110#discussion_r1797773681
I already sent a PR for "when implementing the handling of the received values", and I'm wondering whether we need those macros at all (as outlined in that comment).

Reply:
No, haven't seen it before, but now ;-)